### ISSUE TITLE
default action for illegal characters in path name

### ISSUE THREAD

**Issue by wsbunce** [unsound/hfsexplorer#6]
I love your program and use it to extract user files from Time Machine backups (MAC -> WIN).
I have one request for a enhancement:
When the program hits a path with illegal characters for a Windows file system it stop to request user input.  So far I have always clicked “auto rename”.  It would save a lot of time if this could be set as the default action for all further encounters so that I don’t have to sit at the keyboard for hours.
Thanks for your consideration of this request.

**Comment by MarjaE2** [comment]
Agreed. Because I have to click "auto-rename" every few minutes, I can't just let this run overnight. It would be much more helpful to let users auto-auto-rename, and include a log of the changes at the end.

**Comment by unsound** [comment]
I think this is basically a duplicate of #8 though that one is more specific about the functionality needed. I'm going to close this and then we can track this feature in #8 instead. If you disagree, please reopen. :)
